src/styles/abstracts/_variables.scss

```
@@ -5,6 +5,7 @@ $background: #c0c0c0;
 $light: #fff;
 $dark: #000;
 $shuttle-gray: #5A6570;
```

Please use lower case instead. 
Ref: https://stackoverflow.com/questions/32695983/is-there-a-reason-to-use-uppercase-letters-for-hexadecimal-css-color-values

```suggestion
$shuttle-gray: #5a6570;
```

---

I didn't add this line, but ok to modify it.